biolink-model.yaml

```diff
@@ -2303,6 +2459,22 @@ slots:
     mappings:
       - wgs:long
 
+  affiliation:
```

Comment:
Consider that Affiliations are time bound things, and we may want to be able to describe when someone's affiliation began, and/or ended. We may also want to describe the nature of a person's affiliation  (their status or role in the affiliated organization), which may change over time.  If these are relevant use cases for us, we may eventually need to consider a richer model of Affiliation as a class/object in the model rather than just a multivalued slot taking a curie.
Happy to raise this issue in a separate ticket.

Reply:
Hi again @mbrush, excellent point. I don't know what the Translator use cases might specifically be but I sense that there will be some relevant ones, as we scientists do move around so often given our professional career ambitions!  

I think that Chris M. would be very happy to see this concern developed as a separate ticket since it is not really central to the 'publications' model itself.  Please go ahead and raise the ticket (you can cross reference the  issue https://github.com/biolink/biolink-model/issues/384 and this PR https://github.com/biolink/biolink-model/pull/481 as partially motivating contexts for managing affiliations).

Perhaps, for the time being in this PR, I will simply specify a straw man 'affiliation' proxy category class as a placeholder for elaboration of the affiliation concept, then ponder how best to embed this in the publications use cases(?). I'm not sure more than one affiliation should be modelled as `association` edges or just as a list of "affiliation ids".

In the latter case, not sure how specific instances of affiliation for an individual be identified? How might chronological order be enforced? 

Off the top of my head, if we answer the first question, then we could perhaps vanquish the second by modelling a chronological sequence of affiliations by a field "previous affiliation" pointing to the most recent previous instance. However, what if an individual has multiple concurrent affiliations.  How do we model that: a web of professional intrigue, LOL?

Hmm... also, affiliations are probably 'context specific' for publications, in that, in any given instance of publication, a given author might boast one affiliation (for publication purpose) out of several ones which may effectively have, or may list multiple concurrent affiliations.

This last point actually suggests to me that the 'effective affiliation' of, say, an 'author' should actually rather be annotated on the `publication to agent association` edge class in a publications knowledge graph, not (just) the terminal `agent' node of such an association.

Thanks again @mbrush for your feedback on this!